templates/package/content/generic.tmpl

```diff
@@ -4,7 +4,8 @@
 		<div class="ui form">
 			<div class="field">
 				<label>{{svg "octicon-terminal"}} {{.locale.Tr "packages.generic.download"}}</label>
-				<div class="markup"><pre class="code-block"><code>curl {{AppUrl}}api/packages/{{.PackageDescriptor.Owner.Name}}/generic/{{.PackageDescriptor.Package.Name}}/{{.PackageDescriptor.Version.Version}}/{{(index .PackageDescriptor.Files 0).File.Name}}</code></pre></div>
+				<div class="markup"><pre class="code-block"><code>{{range .PackageDescriptor.Files}}curl {{AppUrl}}api/packages/{{$.PackageDescriptor.Owner.Name}}/generic/{{$.PackageDescriptor.Package.Name}}/{{$.PackageDescriptor.Version.Version}}/{{.File.Name}}
```

Comment:
1. Does that copy the new line?
2. Do I see it correctly that this means that there will be a trailing empty line in the code block?

Reply:
1. yes
2. Looks like this:
![grafik](https://user-images.githubusercontent.com/1666336/186014839-fa72f143-599f-4d44-afe5-1059ce56304e.png)

If there is a better way to do this I'm happy to change it.

Reply:
Ah, does not appear to be present. Great.